pkg/metrics/grafana/tiproxy_summary.json

```diff
@@ -638,10 +638,10 @@
                "steppedLine": false,
                "targets": [
                   {
-                     "expr": "sum(rate(tiproxy_session_query_total{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\"}[1m])) by (backend)",
+                     "expr": "label_replace(sum(rate(tiproxy_session_query_total{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\"}[1m])) by (backend, instance), \"backend\", \"$1\", \"backend\", \"(.+-tidb-[0-9]+).*peer.*.svc.*\")",
```

Comment:
If there are M tiproxy and N tidb, there will be M*N items? That will be too many.

Reply:
You have that natively because it is reported by different tiproxy. They will have multiple lines on grafa because labels of tiproxy instances are different on cloud.

That is the also case of "tiproxy_balance_b_conn" or so.

One can select a specific tiproxy instance to simplify the graph.

Reply:
I recommend to adap the graph to multiple instances. And provude suitable selector, since we cant really controll label of metrics on cloud.

But we could also do some hack like within this PR, to aggregate ignoring difference of different tiproxy instances.

Reply:
Anyway, it is reverted back to the original for now.